src/compositemark/common.ts

```diff
@@ -58,7 +58,7 @@ export function filterTooltipWithAggregatedField<F extends Field>(
 } {
   const {tooltip, ...filteredEncoding} = oldEncoding;
   if (!tooltip) {
-    return {filteredEncoding: oldEncoding};
+    return {filteredEncoding};
```

Comment:
Why this this change related to "fix: don't add guide properties to tooltips"?

Reply:
It's not. It just makes the code less confusing.